packages/java/tests/spring/react-signals/frontend/routes.tsx

```diff
@@ -1,6 +1,7 @@
 import { createBrowserRouter } from 'react-router-dom';
 import MainLayout from './MainLayout.js';
 import BasicSignalView from './_views/BasicSignalView.js';
+import SharedNumberSignal from './_views/SharedNumberSignal.js';
```

Comment:
I guess, we can already rename it: `_views` -> `views`. The problem why we had to rename it has already gone